src/rules/converters/eslint-plugin-react/jsx-ban-elements.ts

```diff
@@ -0,0 +1,38 @@
+import { RuleConverter } from "../../converter";
+
+export const convertJsxBanElements: RuleConverter = (tslintRule) => {
+    return {
+        rules: [
+            {
+                ...collectArguments(tslintRule.ruleArguments),
+                ruleName: "react/forbid-elements",
+            },
+        ],
+        plugins: ["eslint-plugin-react"],
+    };
+};
+
+const collectArguments = (ruleArguments: any[]) => {
+    if (ruleArguments.length === 0) {
+        return undefined;
+    }
+
+    const forbidArr: any[] = [];
+    for (const element of ruleArguments) {
```

Comment:
Mild style nitpicking here: this `for` loop has the sole purpose of pushing to an array, so I think it'd be a little cleaner to use a `.map` instead.

```ts
const forbiddenElements = ruleArguments.map((ruleArgument) => {
    return {
        element: element[0],
        ...(element.length === 2 && { message: element[1] })
    };
});
```

_(also, `===` instead of `==` please)_